prophy/composite.py

```diff
@@ -584,3 +594,12 @@ def __init__(cls, name, bases, attrs):
             extend_union_descriptor(cls, descriptor)
             add_union_properties(cls, descriptor)
         super(union_generator, cls).__init__(name, bases, attrs)
+
```

Comment:
Use only single blank line between two blocks of code (function and function, class and class, etc.).